lib/qrstorage_web/templates/qr_code/download.html.heex

```diff
@@ -2,7 +2,17 @@
     <div id="canvas" data-color={@qr_code.color} data-url={@qr_code.text}></div>
 <% else %>
     <div id="canvas" data-color={@qr_code.color} data-url={Routes.qr_code_url(@conn, :show, @qr_code)}></div>
+    <%= if get_flash(@conn, :admin_url_id) do %>
```

Comment:
Just to be sure. It is intended that qr codes with type `:link` cannot be deleted. I would have expected to work for link qr codes as well

Reply:
This is fine, they get deleted automatically after one hour. We just need to store them to show the QR code.